parkings/importers/geojson_parking_areas.py

```diff
@@ -0,0 +1,96 @@
+import json
+import logging
+import time
+
+from django.db import transaction
+
+from .geojson_importer import GeoJsonImporter
+from parkings.models import EnforcementDomain, ParkingArea
+
+logger = logging.getLogger(__name__)
+
+
+class ParkingAreaImporter(GeoJsonImporter):
+    def __init__(self, file_path, overwrite=False):
+        self.file_path = file_path
+        self.overwrite = overwrite
+        self.refusals = 0
+        self.overwrites = 0
+        self.created = 0
+        self.srid = None
+    
+    def import_areas(self):
+        start = time.time()
+
+        area_dicts = self.read_and_parse(self.file_path)
+
+        self._save_areas(area_dicts)
+
+        if self.refusals:
+            logger.warning(
+                'Refused to overwrite data for %s areas because'
+                ' --overwrite is not set.' % self.refusals
```

Comment:
Use the args passed into logger instead of the in-place percentage sign:
```python
logger.warning('... message %s', param)  # i.e. this
logger.warning('... message %s' % param)  # not this
```

Reply:
and same applies to all of the logger calls here